helm/openig/values.yaml

```diff
@@ -28,20 +28,16 @@ config:
 # Tomcat JVM options.
 catalinaOpts: "-Xmx512m"
   
-# If you set this to true, you must have a TLS secret with the same name as the FQDN
+# TLS is enabled by default. See TLS section for configuration options
```

Comment:
See my suggestion in the README for replacing this comment.

Reply:
done